apps/web/src/app/(unauthenticated)/layout.tsx

```diff
@@ -8,6 +9,13 @@ type UnauthenticatedLayoutProps = {
   children: React.ReactNode;
 };
 
+export const metadata: Metadata = {
+  title: {
+    template: '%s | Documenso',
+    default: 'Documenso',
+  },
+};
```

Comment:
Can this be extracted to the root layout.tsx?

Reply:
Yeah,definitely.